src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -736,8 +764,16 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
 
             if (varTypeIsFloating(simdBaseType))
             {
-                op2 = impSIMDPopStack(retType);
-                op1 = impSIMDPopStack(retType);
+                CORINFO_ARG_LIST_HANDLE arg1     = sig->args;
+                CORINFO_ARG_LIST_HANDLE arg2     = info.compCompHnd->getArgNext(arg1);
+                var_types               argType  = TYP_UNKNOWN;
+                CORINFO_CLASS_HANDLE    argClass = NO_CLASS_HANDLE;
+
+                argType = JITtype2varType(strip(info.compCompHnd->getArgType(sig, arg2, &argClass)));
+                op2     = getArgForHWIntrinsic(argType, argClass);
+
+                argType = JITtype2varType(strip(info.compCompHnd->getArgType(sig, arg1, &argClass)));
+                op1     = getArgForHWIntrinsic(argType, argClass);
```

Comment:
This fixes a pre-existing assert that was being triggered by `operator /(Vector128<T> vector, T scalar)`